### PR TITLE
Feature/pdct 774 remove csvs from backends pipeline trigger

### PR DESCRIPTION
**Pull request by olaughter**
# Description
Thanks to the admin service we no longer need to pass csv's into the pipeline trigger endpoint. This means a whole swathe of code is no longer used. Unpicking this means we can simplify the trigger and no longer run some unnecessary and cumbersome tests (A minute and a half! Which is half the unit tests!).

Unfortunately, pulling on the code-deletion thread revealed a place the old ingest code was in fact still used in two places:
- A handful of minor components where used for the app, so have been preserved
- The whole process was actually being used for test setup

The test setup one presents a problem. But also points to a wider issue beyond the scope of this humble change, that we have multiple different test setup functionalities across our test suite. For now, I have simply moved the ingest code into tests (Which will make it easier to clean up and means we can still turn off the tests for it). I've also added a task to our linear board to [consolidate all our test setup strategies](https://linear.app/climate-policy-radar/issue/PDCT-799/consolidate-test-setup-in-the-backend) (and move away from inline csv data fixtures 🙀) so that this can be flollowed up on in the future

## Type of change

Please select the option(s) below that are most relevant:

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change

## How Has This Been Tested?

Please describe the tests that you added to verify your changes.

## Reviewer Checklist

- [x] The PR represents a single feature (small driveby fixes are also ok)
- [x] The PR includes tests that are sufficient for the level of risk
- [x] The code is sufficiently commented, particularly in hard-to-understand areas
- [x] Any required documentation updates have been made
- [x] Any TODOs added are captured in future tickets
- [x] No FIXMEs remain

### CODE REVIEW

File: app/api/api_v1/routers/cclw_ingest.py
Comment (diversemix) [inline app/api/api_v1/routers/cclw_ingest.py:1]: Worth renaming this file? `pipeline_trigger.py` ?

File: app/main.py [hunk 1]
```diff
@@ -132,7 +131,6 @@ async def root():
 # Create an admin router that is a combination of:
 admin_router = APIRouter()
 admin_router.include_router(cclw_ingest_router)
```
Comment (diversemix) [inline app/main.py:133]: yeah this would be good to be renamed

**Review by diversemix — APPROVED** [review]
lgtm - the suggested rename can always get done as part of the next stage of tidy up :crossed_fingers: it happens :smile:

**Comment by olaughter** [comment]
> lgtm - the suggested rename can always get done as part of the next stage of tidy up 🤞 it happens 😄

Good shout, changing now in case we never get to round two of cleanup 😅

**Pull request merged**